site/content/docs/5.2/about/support-bootstrap.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: docs
+title: Support Bootstrap
+description: Show your support through donations and sponsorships to help maintaining Bootstrap
+group: about
+toc: true
+---
+
+Through donations and sponsorships we are able to maintain & improve Bootstrap. Feel free to show your support on our [Open Collective page](https://opencollective.com/bootstrap).
+
+## Sponsors
+
+{{< sponsors.inline >}}
+<div id="sponsorList" class="d-flex flex-wrap mx-n2 text-center"></div>
+{{< /sponsors.inline >}}
+
+## Backers
+
+{{< backers.inline >}}
+<div id="backerList" class="d-flex flex-wrap mx-n1 text-center"></div>
+{{< /backers.inline >}}
+
+## Services
+
+{{< services.inline >}}
+<div class="d-flex mx-n3 flex-wrap">
+  {{- range (index $.Site.Data "services") }}
+    <div class="m-3 position-relative">
+      {{ if .website -}}
+        <a href="{{ .website }}" class="stretched-link text-reset" title="{{ .name }}" target="_blank" rel="nofollow noopener noreferrer">
+      {{ end -}}
+      <img src="{{ printf ("/docs/%s/assets/img/services/%s") $.Site.Params.docs_version .image }}" alt="{{ .name }}" class="mh-100 mw-100">
+      {{ if .website -}}
+        </a>
+      {{ end -}}
+    </div>
+  {{ end -}}
+</div>
+
+<script>
+  (function () {
+    'use strict'
+
+    var backerDisplayed = 10
+
+    function displaySponsors(sponsorList) {
+      var sponsorListEl = document.getElementById('sponsorList')
+      var output = []
+
+      sponsorList.forEach(function (sponsor) {
+        var sponsorAccount = sponsor.fromAccount
+
+        output.push(
+          '<div class="m-2 position-relative">',
+          ' <div class="img-thumbnail mx-auto d-flex align-items-center justify-content-center overflow-hidden sponsor">',
+          '  <img class="img-fluid d-block" src="' + sponsorAccount.imageUrl + '" alt="' + sponsorAccount.name + '">',
```

Comment:
And below
```suggestion
          `  <img class="img-fluid d-block" src="${sponsorAccount.imageUrl}" alt="${sponsorAccount.name}">`,
```

Reply:
Fixed in https://github.com/twbs/bootstrap/pull/28744/commits/9f78df138c53f291efff4640f842f78200a1562f. Resolving this conversation.